docs-additonal/development-setup.md

```diff
@@ -0,0 +1,59 @@
+# Development Setup for Cobbler-Web
+
+## Basic setup
+
+This will give you a setup of both repositories of the main Git branches.
+
+1. Clone Cobbler: `git clone git@github.com:cobbler/cobbler.git`
+2. Clone Cobbler Web: `git clone git@github.com:cobbler/cobbler-web.git`
+3. Go into the `cobbler` directory and execute the following steps:
+   - Build the Docker image: `docker build -f docker/testing/testing.dockerfile -t cobbler-testing .`
+   - Run the built image: `docker run -d --name cobbler-testing -v $PWD:/code cobbler-testing`
+   - Inspect the container and its IP address: `docker inspect cobbler-testing | grep "IPAddress"`
+4. Go into the `cobbler-web` directory and follow these steps:
+   - Adjust the file `projects/cobbler-frontend/proxy.conf.json` and in the line with `target` replace the string
```

Comment:
incorrect proxy path, should be projects/cobbler-frontend/src/proxy.conf.json

Reply:
Will be fixed by the next commit.